docs/source/guides/staking_guide.rst

```diff
@@ -302,6 +300,54 @@ allow **re-staking** to be disabled until the release period begins, even if you
 No action is needed to release the re-staking lock once the release period begins.
 
 
+Modifying Active Stakes
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Several administrative operations can be performed on active stakes:
+
++----------------------+-------------------------------------------------------------------------------+
+| Action               |  Description                                                                  |
++======================+===============================================================================+
+|  ``divide``          | Create a new stake from part of an existing one                               |
++----------------------+-------------------------------------------------------------------------------+
+|  ``prolong``         | Prolong an existing stake's duration                                          |
++----------------------+-------------------------------------------------------------------------------+
+|  ``winddown``        | Manage winding down of stakes                                                 |
++----------------------+-------------------------------------------------------------------------------+
+
+
+Divide
+******
+
+To divide an existing stake into discrete substakes:
+
+.. code:: bash
+
+    (nucypher)$ nucypher stake divide
+
+
+Prolong
+*******
+
+To prolong an existing stake's duration:
+
+.. code:: bash
+
+    (nucypher)$ nucypher stake prolong
+
+
+Wind Down
+**********
+
+To set the winding down time of an existing stake:
```

Comment:
I'm not sure this is a correct description. The winddown time is equal to the stake duration. Running `nucypher stake winddown` starts the winddown but it doesn't set the time. At least, that's my understanding
```suggestion
To start winding down an existing stake:
```